examples/crowdfunding/scripts/createApp.js

```diff
@@ -0,0 +1,103 @@
+const { executeTransaction, TransactionType, SignType, update } = require('algob');
+const { decodeAddress } = require('algosdk');
+
+/**
+* Description: Converts Integer into Bytes Array
+*/
+function getInt64Bytes (x) {
```

Comment:
I think we are using this function in other examples as well. If so, could we move somewhere to common and reuse?

Reply:
yes